invenio_rdm_records/config.py

```diff
@@ -148,6 +148,37 @@ def always_valid(identifier):
 RDM_ALLOW_RESTRICTED_RECORDS = True
 """Allow users to set restricted/private records."""
 
+#
+# Record deletion by users
+#
+RDM_REQUEST_RECORD_DELETION_ENABLED = False
+"""Allow users to request record deletion."""
+RDM_IMMEDIATE_RECORD_DELETION_ENABLED = False
+"""Allow users to immediately delete records."""
+
+RDM_RECORD_DELETION_POLICY = RDMRecordDeletionPolicy
+"""Policy class which evaluates whether a record can be deleted by a user."""
+RDM_IMMEDIATE_RECORD_DELETION_GRACE_PERIOD = timedelta(days=30)
+"""Grace period for immediate record deletion."""
+
+RDM_RECORD_DELETION_CHECKLIST = [
+    {
+        "label": "I want to change the metadata (title, description, etc)",
+        "message": "You can edit the record <a href='/'>here</a>",
```

Comment:
should the label or the message be translatable?

Reply:
thanks!